index.php

```diff
@@ -0,0 +1,2 @@
+<?php
+header('Location: index.html');
```

Comment:
Really? You cannot host a static site on Dokku I don't believe you :stuck_out_tongue: just need nginx to do this I am sure :stuck_out_tongue: